components/CartBox.php

```diff
@@ -95,6 +100,7 @@ protected function prepareVars()
         $this->page['cartBoxTimeFormat'] = $this->property('cartBoxTimeFormat');
         $this->page['pageIsCart'] = $this->property('pageIsCart');
         $this->page['pageIsCheckout'] = $this->property('pageIsCheckout');
+        $this->page['hideFreeOptionPrices'] = $this->property('hideFreeOptionPrices');        
```

Comment:
For readability, change `hideFreeOptionPrices` to `hideZeroOptionPrices` and cast the value to bool with `...= (bool)$this->property('hideFreeOptionPrices')`.

Reply:
Done